cs/shipdesign.go

```diff
@@ -278,6 +281,16 @@ func ComputeShipDesignSpec(rules *Rules, techLevels TechLevel, raceSpec RaceSpec
 		InnateScanRangePenFactor: hull.InnateScanRangePenFactor,
 	}
 
+	var err error
+	spec.Cost, err = c.GetDesignCost(rules, techLevels, raceSpec, design)
+	if err != nil {
+		log.Warn().
```

Comment:
ComputeShipDesignSpec should be updated to return an error instead of logging a warning.